CHANGELOG

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 
 ## [Unreleased]
+### Added
+- is_open method on Alert model
+- Send resolution notice(s) if not Alert.is_open()
```

Comment:
Changelog doesn't need to get this technical. I want it to be more for what effects our end users will notice. So here I would just say `Send our resolution notices if all needs have been marked resolved`.